dashboard/src/stores/package.ts

```diff
@@ -17,6 +17,11 @@ export const usePackageStore = defineStore("package", {
 
     // A list of packages shown during searches.
     packages: [] as Array<api.EnduroStoredPackageResponseBody>,
+
+    // User-interface interactions between components.
+    ui: {
+      download: new UIRequest(),
+    },
```

Comment:
I guess we could use and watch a property directly, what's the reason for a class with a property and different methods?